Resources/config/admin.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <!-- ARTICLE -->
+        <parameter key="sonata.article.admin.article.class">Sonata\ArticleBundle\Admin\ArticleAdmin</parameter>
+        <parameter key="sonata.article.admin.article.controller">SonataArticleBundle:ArticleAdmin</parameter>
```

Comment:
I re-post here.

For class parameter, it should be removed: http://symfony.com/doc/current/best_practices/business-logic.html#service-no-class-parameter

For the other parameters, I don't know. As a end-user bundle, I would prefer configuration instead.

Using configuration permit to have strict argument check.

What do you think @rande @sonata-project/contributors?